scripts/oneoff/export-service-categories.py

```diff
@@ -2,7 +2,7 @@
 """Usage: export-service-categories.py <g-cloud-version> [--output-folder=OUTPUT] [--stage=STAGE]
 
 Basic services export, for use by the DM performance analyst.
-Outputs a .tsv file per lot to the given --output-folder, with columns as follows:
+Outputs a .csv file per lot to the given --output-folder, with columns as follows:
  - supplierId
  - dunsNumber
  - supplierName
```

Comment:
Should we add serviceDescription to this list?

Reply:
Yes we should!